### DISCUSSION TITLE
Senior Azure Cloud Engineer wanted!

### DISCUSSION THREAD

**Discussion by miqm** [AzureCommunityPL/group#218] [Oferty pracy]
Perfect fit for the role:
☁️   Solid experience with Azure services, Docker, and cloud environments in general;
🦾  Experience in configuring and improving Infrastructure as Code (ARM Templates/Terraform);
💻  Solid practice with management and automation of CI/CD pipelines (Jenkins, CircleCI, Azure DevOps);

Salary: 11 700 - 17 300 PLN Gross (UoP) | 14 000 - 20 700 PLN + VAT (Contractor)
Location: Kraków Poznań Warszawa Gdańsk Wrocław Białystok Katowice Łódź or remotely

Details: https://hubs.ly/H0GGk1W0

**Comment by kaluzaaa** [comment]
@miqm popraw post do reguł: 
![image](https://user-images.githubusercontent.com/4700951/108063640-cdae0200-705b-11eb-810d-e318a4bbc8c8.png)

**Reply by miqm** [reply]
w linku są, ale rozumiem, że referencja nie wystarcza :)
